.github/CODERABBIT_INSTALLATION.md

```
@@ -0,0 +1,159 @@
+
+**Status:** Installation Required
+**Priority:** HIGH
+**Estimated Time:** 5 minutes
+
+---
+
+## Installation Steps
+
+### 1. Install CodeRabbit GitHub App
+
+Visit: **https://github.com/apps/coderabbitai**
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)